tools/src/main/java/org/apache/rocketmq/tools/command/topic/UpdateTopicSubCommand.java

```diff
@@ -169,7 +169,7 @@ public void execute(final CommandLine commandLine, final Options options,
                     }
                     defaultMQAdminExt.createOrUpdateOrderConf(topicConfig.getTopicName(),
                         orderConf.toString(), true);
-                    System.out.printf(String.format("set cluster orderConf. isOrder=%s, orderConf=[%s]",
+                    System.out.printf("%s", String.format("set cluster orderConf. isOrder=%s, orderConf=[%s]",
```

Comment:
Remove inner `String.format` will be more clear

Reply:
Right,and this problem has been fixed.